frontend/packages/ceph-storage-plugin/src/components/dashboards/ocs-dashboards.tsx

```diff
@@ -30,17 +32,32 @@ const OCSDashboardsPage: React.FC<DashboardsPageProps> = ({ match, kindsInFlight
   const dynamicTabExtensions = useExtensions<DynamicDashboardsTab>(isDynamicDashboardsTab);
   const dynamicCardExtensions = useExtensions<DynamicDashboardsCard>(isDynamicDashboardsCard);
 
-  const pluginPages = React.useMemo(
-    () =>
-      getPluginTabPages(
-        [...tabExtensions, ...dynamicTabExtensions],
-        cardExtensions,
-        dynamicCardExtensions,
-        'storage',
-        'persistent-storage',
-      ),
-    [tabExtensions, dynamicTabExtensions, cardExtensions, dynamicCardExtensions],
-  );
+  const isExternalOcs = useFlag(OCS_INDEPENDENT_FLAG);
+  const isInternallOcs = useFlag(OCS_FLAG);
+
+  const pluginPages = React.useMemo(() => {
+    const allTabs: (DashboardsTab | DynamicDashboardsTab)[] = [
+      ...tabExtensions,
+      ...dynamicTabExtensions,
+    ];
+    /** firstTabId === 'independent-dashboard', if it is an external mode OCS cluster.
+     *  firstTabId === 'persistent-storage', if it is not an external mode OCS cluster (internal mode).
+     *  firstTabId === 'object-service', if NooBaaSystem is there, but, StorageCluster is not.
+     */
```

Comment:
@bipuladh Assuming this storage-> overview dashboard will be hidden when ODF is installed. This change is still required for OCS 4.8 and OCP 4.9 users.

Reply:
Need to fix in the new upcoming storage system dashboard